views/legalPages/terms.ejs

```diff
@@ -7,17 +7,37 @@
 <h1 class="heading-xlarge">Terms and conditions</h1>
 
 <div class="column-two-thirds">
-    <p>By using this legalisation service you agree to our <a href="privacy-policy"  rel="external" target="_blank">privacy policy<span class="visually-hidden"> opens in new window</span></a> and to these terms and conditions. Please read them carefully.</p>
+    <ul class="contents-links">
+        <li class="contents-link__wrapper">
+            <a href="#general" class="content-link__text govuk-link">General</a>
+        </li>
+        <li class="contents-link__wrapper">
+            <a href="#applicable-law" class="content-link__text govuk-link">Applicable law</a>
+        </li>
+        <li class="contents-link__wrapper">
+            <a href="#how-to-use-this-service" class="content-link__text govuk-link">How to use this service responsibly</a>
+        </li>
+        <li class="contents-link__wrapper">
+            <a href="#requirements-for-legalisation" class="content-link__text govuk-link">Requirements for legalisation</a>
+        </li>
+        <li class="contents-link__wrapper">
+            <a href="#disclaimer" class="content-link__text govuk-link">Disclaimer</a>
+        </li>
+    </ul>
+
+    <h2 id="general" class="heading-medium">General</h2>
 
-    <h2 class="heading-medium">General</h2>
+    <p>By using this legalisation service you agree to our <a class="govuk-link" href="privacy-policy"  target="_blank">privacy policy</a> and to these terms and conditions. Please read them carefully.</p>
```

Comment:
https://design-system.service.gov.uk/styles/typography/#opening-links-in-a-new-tab

opening in a new tab is an accessibility issue, so avoid if possible, however if you _need_ to open in a new tab add rel="noreferrer noopener"

Reply:
Fair, I'll stick to not opening things in a new tab, although I heard the `rel="noreferrer noopener"` addition wasn't needed nowadays for newer sights mainly because of ssl 🤷

Reply:
nope, as per OWASP you can still do the attack even if if the target site is served over https https://owasp.org/www-community/attacks/Reverse_Tabnabbing

Reply:
ah right, you learn something new everyday